bundles/binding/org.openhab.binding.zwave/database/ge/ZW4005.xml

```diff
@@ -42,9 +41,9 @@
 				<Value>1</Value>
 				<Label lang="en">Yes</Label>
 			</Item>
-			<Help lang="en">Change the top of the switch to OFF and the bottom of the switch to ON. Note: If you invert the switches and also install the product upside down, remember the load will now be controlled by the right, not the left switch.</Help>
+			<Help lang="en"><![CDATA[Change the top of the switch to OFF and the bottom of the switch to ON. <br>
+			Note: If you invert the switches and also install the product upside down, remember the load will <br>
```

Comment:
Same comment here - the BR shouldn't be on the end of this line. The first line is fine, but not the second one as this is all one sentence and it should be up to the GUI to display this as it sees fit.